tests/python/LookTest.py

```diff
@@ -1,28 +1,186 @@
+        """
+
+        # Default initialized transform value is None
+        self.assertIsNone(self.look.getTransform())
+
+        exp_tr = OCIO.ExponentTransform()
+        exp_values = [0.1, 0.2, 0.3, 0.4]
+        exp_tr.setValue(exp_values)
+        self.look.setTransform(exp_tr)
+        oet = self.look.getTransform()
+        self.assertListEqual(oet.getValue(), exp_values)
+
+        # Wrong type tests.
+        for invalid in (OCIO.ALLOCATION_UNIFORM, 1):
+            with self.assertRaises(TypeError):
+                self.look.setTransform(invalid)
+
+    def test_inverse_transform(self):
+        """
+        Test the setInverseTransform() and getInverseTransform() methods.
+        """
+
+        # Default initialized inverse transform value is None
+        self.assertIsNone(self.look.getInverseTransform())
+
+        exp_tr = OCIO.ExponentTransform()
+        exp_values = [-0.1, -0.2, -0.3, -0.4]
+        exp_tr.setValue(exp_values)
+        self.look.setInverseTransform(exp_tr)
+        inv_oet = self.look.getInverseTransform()
+        self.assertListEqual(inv_oet.getValue(), exp_values)
+
+        # Wrong type tests.
+        for invalid in (OCIO.ALLOCATION_UNIFORM, 1):
+            with self.assertRaises(TypeError):
+                self.look.setInverseTransform(invalid)
+
+    def test_constructor_with_keyword(self):
+        """
+        Test Look constructor with keywords and validate its values.
+        """
+
+        # With keywords in their proper order.
+        exp_tr = OCIO.ExponentTransform()
+        inv_exp_tr = OCIO.ExponentTransform()
+        look = OCIO.Look(name='coollook',
+                         processSpace='somespace',
+                         transform=exp_tr,
+                         inverseTransform=inv_exp_tr,
+                         description='this is a test')
+
+        self.assertEqual(look.getName(), 'coollook')
+        self.assertEqual(look.getProcessSpace(), 'somespace')
+        self.assertIsInstance(look.getTransform(), type(exp_tr))
+        self.assertIsInstance(look.getInverseTransform(), type(inv_exp_tr))
+        self.assertEqual(look.getDescription(), 'this is a test')
+
+        # With keyword not in their proper order.
+        exp_tr2 = OCIO.ExponentTransform()
+        inv_exp_tr2 = OCIO.ExponentTransform()
+        look2 = OCIO.Look(inverseTransform=inv_exp_tr,
+                          description='this is a test',
+                          name='coollook',
+                          processSpace='somespace',
+                          transform=exp_tr)
+
+        self.assertEqual(look2.getName(), 'coollook')
+        self.assertEqual(look2.getProcessSpace(), 'somespace')
+        self.assertIsInstance(look2.getTransform(), type(exp_tr2))
+        self.assertIsInstance(look2.getInverseTransform(), type(inv_exp_tr2))
+        self.assertEqual(look2.getDescription(), 'this is a test')
+
+    def test_constructor_without_keyword(self):
```

Comment:
Recommend using `test_constructor_positional` or `test_constructor_with_positional` instead of `test_constructor_without_keyword`.

Reply:
Resolved in 2b67585.